forward/fwd.go

```diff
@@ -327,8 +327,10 @@ func (f *httpForwarder) serveWebSocket(w http.ResponseWriter, req *http.Request,
 	}
 	go replicate(targetConn, underlyingConn, "backend", "client")
 	go replicate(underlyingConn, targetConn, "client", "backend")
-	err = <-errc
-	log.Infof("vulcand/oxy/forward/websocket: websocket proxying complete: %v", err)
+	err = <-errc // One goroutine complete
+	log.Infof("vulcand/oxy/forward/websocket: > websocket proxying complete: %v", err)
+	err = <-errc // Both goroutines complete
+	log.Infof("vulcand/oxy/forward/websocket: < websocket proxying complete: %v", err)
```

Comment:
Can you update the text to either indicate a direction that it closed, or just say closed on one way, and closed on second way?

They may not necessary close in the > order and then the < order. That could cause debugging confusion later.